spyder/app/tests/test_mainwindow.py

```diff
@@ -1688,7 +1688,11 @@ def test_troubleshooting_menu_item_and_url(monkeypatch):
 @flaky(max_runs=3)
 @pytest.mark.slow
 def test_help_opens_when_show_tutorial_full(main_window, qtbot):
-    """Test fix for #6317 : 'Show tutorial' opens the help plugin if closed."""
+    """
+    Test fix for spyder-ide/spyder#6317 :
```

Comment:
I think we should settle on a style, and try to use the same style for all inside docstrings.

Since the regex will work well no matter if a dot or colon is added, I guess the best would be to always use periods `.` within docstrings, if the spyder-ide/spyder#NNNN is at the end of a sentence/line